src/Tizen.NUI/src/public/Animation/Animation.cs

```diff
@@ -230,6 +236,10 @@ public AlphaFunction DefaultAlphaFunction
         {
             set
             {
+                if (value is AlphaFunction alpha)
+                {
+                    addAlphaFunctionInList(alpha);
+                }
```

Comment:
AlphaFunction타입의 property의 setter이기 때문에 value가 null이 아닌 이상 AlphaFunction이 아닐수는 없습니다. 그래서 `is AlphaFunction`으로 검사하는것이 이상합니다.
```suggestion
                if (value != null)
                {
                    addAlphaFunctionInList(value);
                }
```